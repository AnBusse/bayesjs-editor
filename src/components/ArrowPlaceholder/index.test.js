import React from 'react';
import { shallow } from 'enzyme';
import ArrowPlaceholder from './index';

const defaultProps = {
  from: { x: 0, y: 0 },
  to: { x: 100, y: 100 },
};

const shallowComponent = (props = {}) => {
  const compProps = { ...defaultProps, ...props };

  return shallow(<ArrowPlaceholder {...compProps} />);
};

describe('ArrowPlaceholder Component', () => {
  it('matches snapshot', () => {
    const component = shallowComponent();

    expect(component).toMatchSnapshot();
  });
});
