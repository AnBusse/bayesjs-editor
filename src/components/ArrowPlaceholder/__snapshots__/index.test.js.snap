// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowPlaceholder Component matches snapshot 1`] = `
<path
  d="M0,0 100,100"
  fill="none"
  markerEnd="url(#triangle)"
  stroke="#333"
  strokeDasharray="5,5"
  strokeWidth="2"
/>
`;
